sw/host/opentitanlib/src/transport/verilator/subprocess.rs

```diff
@@ -35,9 +35,10 @@ pub struct Subprocess {
 impl Subprocess {
     /// Starts a verilator [`Subprocess`] based on [`Options`].
     pub fn from_options(options: Options) -> Result<Self> {
-        let mut command = Command::new(&options.executable);
+        let mut command = Command::new(String::from("nice"));
         let mut args = Vec::new();
-
+        args.push(String::from("-5"));
```

Comment:
Peeking at `man nice`, doesn't `-5` need to be preceded by `-n` or `--adjustment=`?

Reply:
That makes it clearer we're increasing niceness but less concise.

Reply:
Actually, now I'm worried that `nice` might be reading plain `-5` as a parameter and silently ignoring it.

```
% nice -5 bash -c 'echo hello'
hello
% nice -n -5 bash -c 'echo hello'
nice: cannot set niceness: Permission denied
hello
```

Reply:
It works on my machine
`nice -5 yes > /dev/null & htop`

-5 should be equivalent to -n 5 IIRC

Reply:
I'll make it more verbose and readable, I just want to check that it works before I push the change.

Reply:
the verilated model fell out of my cache again.

positive nicing requires lesser permissions

Reply:
That's why I don't just negative nice the harnesses

Reply:
FYI, `nice` prints niceness, so you can experiment with `nice -5 nice` vs `nice -n -5 nice`.

Reply:
nice!

Reply:
> positive nicing requires lesser permissions

Good point. Positive nicing is less favorable to the process, so I think `-5` is the wrong direction.

Reply:
It's misleading so I'm changing it
`nice -5 nice`:5
`nice -n 5 nice`:5

Reply:
it's a different sort of hyphen

Reply:
Oh! I thought it was silently ignoring `-5`, but it actually interpreted it as `-n 5`. Got it.

I think my persistent confusion about this is evidence it should be a verbose `-n 5` :P